mix.exs

```diff
@@ -4,17 +4,20 @@ defmodule Posthog.MixProject do
   def project do
     [
       app: :posthog,
-      version: "0.4.0",
-      elixir: "~> 1.12",
-      start_permanent: Mix.env() == :prod,
       deps: deps(),
       description: description(),
-      package: package()
+      elixir: "~> 1.12",
+      elixirc_paths: elixirc_paths(Mix.env()),
+      package: package(),
+      start_permanent: Mix.env() == :prod,
+      version: "0.4.0"
```

Comment:
I'll need to check whether this won't trip CI/CD and the automatic version bump, but it should be fine